template/packages/tsconfig/nextjs.json

```diff
@@ -13,7 +13,7 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "module": "esnext",
-    "moduleResolution": "bundler",
+    "moduleResolution": "NodeNext",
```

Comment:
I've used recommended tsconfig for nextjs (https://www.npmjs.com/package/@tsconfig/next)

Why do you want to replace it with "NodeNext"?

Reply:
I had precommit errors and found this solution
But now all fine with 'bundler'. So I rolled back to this setting